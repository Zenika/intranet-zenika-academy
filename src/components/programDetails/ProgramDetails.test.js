// import ReactDOM from 'react-dom';
// import renderer from 'react-test-renderer';
// import React from 'react';
// import Enzyme, { shallow } from 'enzyme';
// import axios from 'axios';
// import Adapter from 'enzyme-adapter-react-16';
// import ProgramDetails from './ProgramDetails';

// Enzyme.configure({ adapter: new Adapter() });

// jest.mock('axios');

// describe('ProgramDetails component', () => {
// it.only('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<ProgramDetails />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
// it.only('matches snapshot', () => {
//   const component = renderer.create(<ProgramDetails />);
//   const tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
// });
// it('contains a title "Bienvenue"', () => {
//   const app = shallow(<ProgramDetails />);
//   expect(app.containsMatchingElement(<h1>Bienvenue</h1>)).toEqual(true);
// });
// describe('when rendered', () => {
//   it('should fetch a promotion', () => {
//     const getSpy = jest.spyOn(axios, 'get');
//     shallow(
//       <ProgramDetails />,
//     );
//     expect(getSpy).toBeCalled();
//   });
// });
// });
