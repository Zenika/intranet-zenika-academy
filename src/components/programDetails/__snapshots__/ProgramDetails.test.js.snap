// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgramDetails component matches snapshot 1`] = `
<section
  className="studentHomeSection"
/>
`;
